hack/release-automater/release_overview.py

```
@@ -0,0 +1,369 @@
+
+from typing import List, Dict
+
+REPO_ID = "5fb813954070f53cd79231ff"
+PYXIS_API = "https://catalog.redhat.com/api/containers/v1"
+
+def to_date(value):
+    if not value or value == "N/A":
+        return None
+    try:
+        return parse(value).date()
+    except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)